### PR TITLE
Update strings_tr-TR.i18n.yaml

### PR DESCRIPTION
**Pull request by kyoyacchi**
Updated new strings and fixed some mistranslated strings. + Some grammar..

### CODE REVIEW

**Comment by kyoyacchi** [comment]
I wonder what have i done wrong?

**Comment by nullxception** [comment]
> I wonder what have i done wrong?

looks like you give the "blur_timeline_only:" too much tab/space, not aligned with other section. check the line 180.

File: lib/presentation/i18n/strings_tr-TR.i18n.yaml [hunk 1]
```diff
@@ -171,6 +177,9 @@ settings:
   blur_content:
     title: "Müstehcen içeriği bulanıklaştır"
     desc: "Müstehcen dereceli gönderilen bulanıklaştırılır"
+    blur_timeline_only:
```
Comment (nullxception) [inline lib/presentation/i18n/strings_tr-TR.i18n.yaml:180]: this one need to be aligned with blur_content or ime_incognito

**Review by nullxception — CHANGES_REQUESTED** [review]

**Comment by nullxception** [comment]
I'll merge it anyway since it was a small mistake, Thanks!

**Pull request merged**